app/Main.hs

```diff
@@ -8,20 +9,33 @@ import Data.Monoid ((<>))
 
 import Options.Applicative
 
-import Configuration.Dotenv (loadFile)
-
+import Control.Monad (void)
 import Control.Monad.IO.Class(MonadIO(..))
 
+import System.FilePath ((</>))
+import System.Directory (getCurrentDirectory)
 import System.Process (system)
 import System.Exit (exitWith)
 
+import Configuration.Dotenv
+import Configuration.Dotenv.Types
+
 data Options = Options
-  { files    :: [String]
-  , overload :: Bool
-  , program  :: String
-  , args     :: [String]
+  { program           :: String -- ^ Program to run with the load env variables
+  , args              :: [String] -- ^ Args for the program that is going to be executed
+  , dotenvFile        :: String -- ^ Path for the .env file
```

Comment:
@habibalamin It is related to #41. However, we've been discussing about this change and we want to add a flag (`--safe`) to behave this way. That is why this PR is on hold for now.